ibis/pandas/core.py

```diff
@@ -384,3 +466,38 @@ def execute_and_reset(
     elif isinstance(result, pd.Series):
         return result.reset_index(drop=True)
     return result
+
+
+compute_time_context = Dispatcher(
```

Comment:
Add contract  of this method in documentation.

Reply:
I add docs explaining input params and return types for this function.